homeworks/ReactProject/src/components/router/Pages.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {Switch} from 'react-router';
+import {Route} from 'react-router-dom';
+
+import { LoginPage } from '../../pages/loginPage/LoginPage'
```

Comment:
You can use re-export to avoid code duplicating
```
import { 
  LoginPage,
  CoursesPage,
  CourseManipulatePage,
  ...
} from '../../pages
```

/page/index.js
```
export * from './loginPage';
export * from './coursePage';
...
```

Reply:
general export was added for pages